### ISSUE TITLE
Can this detection library be used for training? 

### ISSUE THREAD

**Issue by xuedue** [1adrianb/face-alignment#287]
I have pip this package, And the effect is very good.

But I want to know if this can be used for training?

For example, I want to predict the landmarks of the generated image and the input image, and calculate the landmark loss to train the network.

**Comment by jkwang28** [comment]
It seems they use the '.detach()' when processing tensors and convert them to numpy.
